### PR TITLE
MES-1190 Manoeuvre fault marking to be limited to a single type of fault

### PR DESCRIPTION
**Pull request by ammarhaiderbjss**
## Pull Request checklist:

- [x] [WIP] tag removed from PR title
- [x] PR has an understandable description

## Git feature branch checklist:

- [x] branch name comply with our branching strategy
- [x] git branch contains relevant JIRA ticket number
- [x] branch rebased against the latest develop
- [x] commits are squashed

## Sign off process checklist:

- [x] Code has been tested manually
- [x] Tests are passing
- [x] PR link added to JIRA
- [x] Reviewers selected in Github
- [x] Any new reusable component/widget added to component library on Confluence

- [x] Screenshot(s) captured on the physical device (if applicable)
- [x] Tested by QA
- [ ] PO's approval

### CODE REVIEW

File: src/components/all-on-one-form-sub-element-hold-no-modal/all-on-one-form-sub-element-hold-no-modal.ts [hunk 1]
```diff
@@ -129,6 +128,7 @@ export class AllOnOneFormSubElementHoldNoModalComponent {
 
   addDangerousFault() {
     if (this.dangerous) return;
+    if (this.isManoeuvre() && (this.faultCounter > 0 || this.serious)) return;
```
Comment (szabolcs-keresztes) [inline src/components/all-on-one-form-sub-element-hold-no-modal/all-on-one-form-sub-element-hold-no-modal.ts]: What happens if somebody goes in and removes the line number 130?

Comment (ammarhaiderbjss) [reply src/components/all-on-one-form-sub-element-hold-no-modal/all-on-one-form-sub-element-hold-no-modal.ts]: 💥

Comment (ammarhaiderbjss) [reply src/components/all-on-one-form-sub-element-hold-no-modal/all-on-one-form-sub-element-hold-no-modal.ts]: would add another dangerous fault into the redux store I would think

Comment (szabolcs-keresztes) [reply src/components/all-on-one-form-sub-element-hold-no-modal/all-on-one-form-sub-element-hold-no-modal.ts]: ahh cool :D

**Comment by ammarhaiderbjss** [comment]
PO's have clarified and approved functionality so testing is sufficient

**Pull request merged**